docs/content/error/$compile/ctxoverride.ngdoc

```diff
@@ -0,0 +1,13 @@
+@ngdoc error
+@name $compile:ctxoverride
+@fullName Context Override
+@description
+
+This error occurs when the security context for a property is defined multiple times under different security contexts.
```

Comment:
Would this just be something such as: "See https://docs.angularjs.org/api/ng/provider/$compileProvider#addPropertySecurityContext"?

Reply:
Not sure what you are asking 😕. If you are asking about the text, it can be whatever you want (as long as there is a link to `addPropertySecurityContext`). 

(If you are asking about how to create the link, it will something like `{@link ng.$compilerProvider#addPropertySecurityContext addPropertySecurityContext()}`.)

Reply:
Was mainly wondering about the `@link` (and couldn't find an example in my 30s attempt) :+1: